L/LibCURL/build_tarballs.jl

```diff
@@ -40,7 +41,7 @@ fi
 make -j${nproc}
 make install
 install_license COPYING
-cp ../cacert.pem $prefix/share/
+cp ../cacert-$cacert_version.pem $prefix/share/cacert.pem
```

Comment:
You can't do this because it's a raw string

Reply:
Ah... duh!